account_voucher_tax/__openerp__.py

```diff
@@ -45,7 +45,8 @@
     "data": [
         "account_tax_view.xml",
         "account_voucher_tax_view.xml",
-        "security/ir.model.access.csv"
+        "security/ir.model.access.csv",
+        "data/ir_actions_server.xml"
```

Comment:
Please add the trailing comma at the end of the new 
element being added,
next time new change will involve only one line.